### PR TITLE
Add one function to ensure default network exists

### PR DESCRIPTION
**Pull request by yalzhang**
Ensure the default network exists and in the expected status.

### CODE REVIEW

File: virttest/utils_net.py [hunk 1]
```diff
@@ -4467,3 +4468,19 @@ def delete_linux_bridge_tmux(linux_bridge_name, iface_name=None, ignore_status=F
     else:
         cmd = 'ip link delete %s' % linux_bridge_name
     return utils_misc.cmd_status_output(cmd, shell=True, verbose=True, ignore_status=ignore_status)
+
+
+def ensure_default_network():
```
Comment (Yingshun) [inline virttest/utils_net.py:4473]: I think utils_net module should be focus on system network, so how about adding it to utils_libvirt.libvirt_network?

Comment (yalzhang) [reply virttest/utils_net.py]: Updated, Thank you!

File: virttest/utils_libvirt/libvirt_network.py [hunk 2]
```diff
@@ -163,3 +163,19 @@ def modify_network_xml(net_dict, testnet_xml):
...
+    """
+    net_state = virsh.net_state_dict()
+    if 'default' not in net_state:
+        # define and start the default network
+        virsh.net_define("/usr/share/libvirt/networks/default.xml",
+                         debug=True, ignore_status=False)
+        virsh.net_start("default", debug=True, ignore_status=False)
+        virsh.net_autostart("default", debug=True, ignore_status=False)
+    elif not net_state["default"].get("active"):
+        virsh.net_start("default", debug=True, ignore_status=False)
```
Comment (chunfuwen) [inline virttest/utils_libvirt/libvirt_network.py:181]: Do we need also apply virsh.net_autostart("default", debug=True, ignore_status=False) after this?

File: virttest/utils_libvirt/libvirt_network.py [hunk 3]
```diff
@@ -163,3 +163,20 @@ def modify_network_xml(net_dict, testnet_xml):
...
+        virsh.net_autostart("default", debug=True, ignore_status=False)
```
Comment (Yingshun) [inline virttest/utils_libvirt/libvirt_network.py:182]: How about changing it like:
```suggestion
    if 'default' not in net_state:
        # define and start the default network
        virsh.net_define("/usr/share/libvirt/networks/default.xml",
                         debug=True, ignore_status=False)
    if not net_state["default"].get("active"):
        virsh.net_start("default", debug=True, ignore_status=False)
        virsh.net_autostart("default", debug=True, ignore_status=False)
```

File: virttest/utils_libvirt/libvirt_network.py (hunk 2 above)
Comment (yalzhang) [reply virttest/utils_libvirt/libvirt_network.py:179]: you are right. Updated.

File: virttest/utils_libvirt/libvirt_network.py (hunk 3 above)
Comment (yalzhang) [reply virttest/utils_libvirt/libvirt_network.py]: Updated, Thank you!

**Review by Yingshun — APPROVED** [review]

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**